prompt/prompt-executor/prompt-executor-llms-all/src/commonMain/kotlin/ai/koog/prompt/executor/llms/all/SimplePromptExecutors.kt

```diff
@@ -44,4 +47,18 @@ public fun simpleGoogleAIExecutor(apiKey: String): SingleLLMPromptExecutor = Sin
  *
  * @param baseUrl url used to access Ollama server.
  */
-public fun simpleOllamaAIExecutor(baseUrl: String = "http://localhost:11434"): SingleLLMPromptExecutor = SingleLLMPromptExecutor(OllamaClient(baseUrl))
\ No newline at end of file
+public fun simpleOllamaAIExecutor(baseUrl: String = "http://localhost:11434"): SingleLLMPromptExecutor = SingleLLMPromptExecutor(OllamaClient(baseUrl))
+
+/**
+ * Creates an instance of `SingleLLMPromptExecutor` with a `BedrockLLMClient`.
+ *
+ * @param awsAccessKeyId Your AWS Access Key ID.
+ * @param awsSecretAccessKey Your AWS Secret Access Key.
+ * @param settings Custom client settings for region and timeouts.
+ */
+public fun simpleBedrockExecutor(
+    awsAccessKeyId: String,
+    awsSecretAccessKey: String,
+    settings: BedrockClientSettings = BedrockClientSettings()
```

Comment:
This doesn't compile, Bedrock client has to be added to the module dependencies

Reply:
It is - the issue is that Bedrock is in `jvmMain` and this code is in `commonMain`. Going to think about it a bit more before changing